src/Iso8601DurationHelper/Duration.cs

```diff
@@ -678,4 +678,32 @@ private enum DurationComponent
             Seconds = Minutes + 1
         }
     }
+
+    /// <summary>
+    /// Extension class to add functionality to <see cref="DateTime"/> and <see cref="DateTimeOffset"/> so that use of <see cref="Duration"/> is similar to using <see cref="TimeSpan"/>
+    /// </summary>
+    public static class DateTimeHelper 
+    {
+        /// <summary>
+        /// Adds a <see cref="Duration"/> value to a <see cref="DateTime"/> instant
```

Comment:
Why "instant"? Just "Adds a Duration value to a DateTime" would be fine, I think. (with the `<see>` tags you already included)

Reply:
Done